### DISCUSSION TITLE
Datatable disappears on Livewire event

### DISCUSSION THREAD

**Discussion by scott80109** [livewire/livewire#4865] [1. Help]
I am trying to build a CRUD system using DataTables. Basically, I'm doing what's in the tutorial at: 

https://shouts.dev/articles/laravel-livewire-crud-with-example

only I am using a DataTable instead of a simple HTML table. So inside of my Livewire component is the DataTable, with "New" and "Update" modals to insert new data, and update existing data. 

The DataTable renders fine upon page load. However, when I open the Create modal and start typing in the form, the DataTable disappears (as soon as any Livewire event is fired). 

Doing some playing around, I emit an event in the component's "updated()" method, and added jQuery code to reload the DataTable. That works fine. However, there must be a better way. I don't want to have to reload the DataTable 100 times every time someone tries to fill out the form. 

Is there a way to prevent the DataTable from disappearing and having to be reloaded? I have tried "wire:ignore" in both the <table> tag, and <div> tag that surrounds the DataTable. That does nothing.

**Comment by denizozturk** [comment]
Can you share the blade codes? It will allow us to help more

For the create form you have to make the inputs `wire:model.defer`, then the component will be refreshed only when submit is triggered. Generate the table with "jquery" when the component is refreshed after submission will solve your problem.

**Reply by scott80109** [reply]
Sorry. I ended up backing Livewire out of my project. It was proving to be too difficult to use alongside jQuery and Datatables.

**Reply by pholophus** [reply]
may I know how you do it? is there a sample code you can show, facing this problem now
